### PR TITLE
Add methods for conversions to SDK maps

### PR DESCRIPTION
**Pull request by kubukoz**
I suppose many people using dynosaur will be doing these at some point, so some place for conversions would be nice (I'm open to suggestions about moving these to some other place).

Should probably get some tests and documentation before merge.

### CODE REVIEW

**Comment by marko-asplund** [comment]
Very nice little util methods and exactly what I'd need! 👍
Are there any blockers to merging this? 🤔

File: modules/core/src/main/scala/DynamoValue.scala [hunk 1]
```diff
@@ -219,6 +219,21 @@ object DynamoValue {
   def b(value: ByteVector): DynamoValue =
     make(_.b(SdkBytes.fromByteArray(value.toArray)))
 
+  /** Builds a DynamoValue from an AWS SDK-compatible attribute map, e.g. from a GetItem response.
+    */
+  def fromAttributeMap(
```
Comment (SystemFw) [inline modules/core/src/main/scala/DynamoValue.scala:224]: should be called `attributeMap` for consistency

File: modules/core/src/main/scala/DynamoValue.scala [hunk 2]
```diff
@@ -219,6 +219,21 @@ object DynamoValue {
...
+      attributes: java.util.Map[String, AttributeValue]
+  ): DynamoValue = make(_.m(attributes))
+
+  /** Converts a DynamoValue to an AWS SDK-compatible attribute map, e.g. for a PutItem operation.
+    * Returns None if the value isn't backed by a map.
+    */
+  def toAttributeMap(
```
Comment (SystemFw) [inline modules/core/src/main/scala/DynamoValue.scala:231]: should be called `attributeMap` for consistency, and be an instance method like the others

**Review by SystemFw — COMMENTED** [review]
Sorry for the delay looking at this PR :)
I'm okay with these being here even though it's easy to do it yourself. Left a couple of suggestions on shape, then I'd like to see docs (and minimal tests) if possible.

**Pull request merged**